cmd/check_operator.go

```diff
@@ -31,6 +31,11 @@ var checkOperatorCmd = &cobra.Command{
 			Image:          operatorImage,
 			EnabledChecks:  engine.OperatorPolicy(),
 			ResponseFormat: defaultOutputFormat,
+			LogFile:        defaultLogFile,
+		}
+
+		if err := initLogger(cfg); err != nil {
+			return fmt.Errorf("unable to initialize the logger for the configuration: %e", err)
```

Comment:
Do we want to create an error in the `errors.go`?

Reply:
Oh! I created one and then forgot to use it. 🤦 I'll fix that!

Reply:
@samira-barouti fixed